test/pico_float_test/custom_double_funcs_test.c

```diff
@@ -236,12 +241,25 @@ int test() {
     test_checki(double2int(-0.75), -1, "double2int5");
     test_checki(double2int(-1.0), -1, "double2int5b");
     // todo test correct rounding around maximum precision
+    test_checki(double2int(2147483646.0), INT32_MAX-1, "double2int6");
     test_checki(double2int(2147483647.0), INT32_MAX, "double2int6");
```

Comment:
You've now used "double2int6" twice, but every other test_check label is unique.